samples/apps/spring-petclinic/.patch/pom.xml.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/pom.xml b/pom.xml
+index ae02829..ef52b15 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -30,6 +30,8 @@
+     <nohttp-checkstyle.version>0.0.4.RELEASE</nohttp-checkstyle.version>
+     <spring-format.version>0.0.27</spring-format.version>
+ 
++    <spring-data-jdbc.version>1.2.1.RELEASE</spring-data-jdbc.version>
++
+   </properties>
+ 
+   <dependencies>
+@@ -46,6 +48,10 @@
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-data-jpa</artifactId>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.boot</groupId>
++      <artifactId>spring-boot-starter-jdbc</artifactId>
++    </dependency>
+     <dependency>
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-web</artifactId>
+@@ -58,6 +64,11 @@
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-thymeleaf</artifactId>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.cloud</groupId>
++      <artifactId>spring-cloud-bindings</artifactId>
++      <version>1.8.0</version>
++    </dependency>
+     <dependency>
+       <groupId>org.springframework.boot</groupId>
+       <artifactId>spring-boot-starter-test</artifactId>
+@@ -86,6 +97,17 @@
+       <artifactId>postgresql</artifactId>
+       <scope>runtime</scope>
+     </dependency>
++    <dependency>
++      <groupId>org.springframework.data</groupId>
++      <artifactId>spring-data-jdbc-core</artifactId>
++      <version>${spring-data-jdbc.version}</version>
++      <exclusions>
++        <exclusion>
++          <groupId>org.springframework</groupId>
++          <artifactId>*</artifactId>
++        </exclusion>
```

Comment:
same question as above?

Reply:
this was needed for `spring-data-jpa` profile used - not needed anymore